packages/compat/package.json

```diff
@@ -76,7 +76,7 @@
         "@solana/addresses": "workspace:*",
         "@solana/keys": "workspace:*",
         "@solana/rpc-types": "workspace:*",
-        "@solana/web3.js": "workspace:*"
+        "@solana/web3.js": "^1"
```

Comment:
The idea here is that we lock this to a public version so that Changesets leaves it alone. This OK, @mcintyre94?

Reply:
Because `@solana/compat` is and it's configured as `fixed`.

* https://github.com/changesets/changesets/pull/507
* https://github.com/changesets/changesets/issues/506
* https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md#fixed-packages

Reply:
I figured out a hack. If you specify the workspace as a relative path it gets ignored by `changeset version`.